### DISCUSSION TITLE
Practica HTML - Pasteleria

### DISCUSSION THREAD

**Discussion by Will0220** [Launch-X-Latam/MisionFrontEnd#250] [General]
Chicos les dejo aca lo que hice de la practica de html. 
(https://github.com/Will0220/Practica-HTML---Pasteleria.git)

Tengo una pregunta en cuanto a subir la pagina por git al dominio, no lo he logrado, si alguno tiene la respuesta me colabora por favor!

**Comment by floressdi** [comment]
Hola!! lo haces de la siguiente manera
 Visual estudio code es de la siguiente manera
1.- Abre una terminal en tu carpeta del proyecto 
Los parentesis no van en la terminal solo los puse para enmarcar lo que se debe poner
    * ver > Terminal
    * En la terminal pones ( git init ) y presionas enter despues de cada comando 
    * luego ( git add . )
    * luego ( git commit -m "Mi primer commit" )
    *  despues ( git remote  add origin  https:Aqui/Va/Tu/Repositiorio/De/github/que/ya debes de tener creado )
    * por ultimo debes fijarte en tu Visual studio code si es rama main o master lo ves en la esquina inferior izquierda abjo del boton de configuracion si es master como en mi caso va el siguiente comando.
    * git push -u origin master   
    * si no es main pues solo lo cambias
    * git push -u origin main 
    * y listo 
    * PD : si no me entendiste igual te dejo una mas explicado es la que yo ocupe
    * https://gist.github.com/cgonzalezdai/cc33db72a6fe5178637aabb562eae35c
